### DISCUSSION TITLE
Is there a way to emit to a specific instance of a component?

### DISCUSSION THREAD

**Discussion by akulmehta** [livewire/livewire#2490] [1. Help]
I want to emit an update on records to a specific instance of a component, which gets repeated multiple times on a page (under different components). Is there a way to fire an update to a specific instance of the component within the same page for e.g. using some sort of key?

**Comment by mattharcourt** [comment]
This might address your issue: https://github.com/livewire/livewire/pull/1181#issuecomment-710216324

**Reply by akulmehta** [reply]
Thanks @mattharcourt.
